notebooks/Fine_tuning_Code_LLM_on_single_GPU.ipynb

```diff
@@ -0,0 +1,5363 @@
+{
```

Comment:
<p>A brief rundown of what we're doing in plain English would be nice. Or even adding comments in the methods would do. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Added comments in the code.